client/src/modules/list/components/Items/ListItem/ListItem.jsx

```diff
@@ -407,6 +388,15 @@ class ListItem extends PureComponent {
       done,
       isNameEdited
     } = this.state;
+    let isNameBusy;
+    let isEdited;
+
+    if (busy) {
+      const { nameBusy, descriptionBusy } = busy;
+
+      isNameBusy = nameBusy;
```

Comment:
Can we use nameBusy directly instead of creating new alias?

Reply:
how can I get access to `nameBusy` variable outside if block without aliasing?

Reply:
You can always do sth like this:
`const { nameBusy, descriptionBusy } = busy || {}`
Do you need also some deafult values? If yes then:
`const { nameBusy = false, descriptionBusy = false } = busy || {}`